mora/service/org.py

```diff
@@ -17,11 +17,16 @@
 
 import enum
 import operator
+import uuid
 
 import flask
 import werkzeug
 
-from . import common, keys
+from mora import lora
+from . import mapping
+from .common import (create_organisationsenhed_payload, _create_virkning,
+                     inactivate_old_interval, set_object_value, update_payload,
+                     ensure_bounds)
```

Comment:
I don't know why, but this style of import always seemed a bit messy to me…

Reply:
I agree, that's why I fixed it. :)

Reply:
Yeah, I noticed that later 🙂